.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
So on any push or PR to master branch the build on pipelines is triggered? Are we going to limit it to only PR or we'll change it later? Or just leave as it is?

Reply:
changed to PR and manual dispatch